src/components/Form/Form.js

```diff
@@ -70,6 +76,7 @@ export class Form extends React.PureComponent {
         {children}
         <Form.Actions
           direction={actionDirection}
+          focusable={actionFocusable}
```

Comment:
Is this doing anything? Form.Actions isn't expecting a new prop, and I took it out when I was playing in Storybook and the buttons still became unfocused.